frontend/assets/javascripts/src/utils/helper.js

```diff
@@ -46,7 +46,16 @@ define(function () {
         return Array.prototype.slice.call(nodeList);
     };
 
+    var extend = function (target, source) {
```

Comment:
As discussed, might be worth pulling this file out into relevant files. Doing quite a lot here. Also worth pulling in `lodash-amd` for `toArray` and `extend`. Not for this PR though.